artifactory/repositories.go

```diff
@@ -97,65 +97,65 @@ func (l LocalRepository) String() string {
+	ComposerRegistryURL               *string                 `json:"composerRegistryUrl,omitempty" xml:"composerRegistryUrl,omitempty"`
+	PyPIRegistryURL                   *string                 `json:"pyPIRegistryUrl,omitempty" xml:"pypi>pyPIRegistryUrl,omitempty"`
+	PyPIRepositorySuffix              *string                 `json:"pyPIRepositorySuffix,omitempty" xml:"pyPIRepositorySuffix,omitempty"`
+	VcsType                           *string                 `json:"vcsType,omitempty" xml:"vcs>type,omitempty"`
+	VcsGitProvider                    *string                 `json:"vcsGitProvider,omitempty" xml:"vcs>git>provider,omitempty"`
+	VcsGitDownloadUrl                 *string                 `json:"vcsGitDownloadUrl,omitempty" xml:"vcs>git>downloadUrl,omitempty"`
+	BypassHeadRequests                *bool                   `json:"bypassHeadRequests,omitempty" xml:"bypassHeadRequests,omitempty"`
+	ClientTLSCertificate              *string                 `json:"clientTlsCertificate,omitempty" xml:"clientTlsCertificate,omitempty"`
+	ExternalDependenciesEnabled       *bool                   `json:"externalDependenciesEnabled,omitempty" xml:"externalDependencies>enabled,omitempty"`
+	ExternalDependenciesPatterns      *[]string               `json:"externalDependenciesPatterns,omitempty" xml:"externalDependencies>patterns>pattern,omitempty"`
+	DownloadRedirect                  *bool                   `json:"downloadRedirect,omitempty" xml:"downloadRedirect>enabled,omitempty"`
+	FeedContextPath                   *string                 `json:"feedContextPath,omitempty" xml:"nuget>feedContextPath,omitempty"`
+	DownloadContextPath               *string                 `json:"downloadContextPath,omitempty" xml:"nuget>downloadContextPath,omitempty"`
+	V3FeedUrl                         *string                 `json:"v3FeedUrl,omitempty" xml:"nuget>v3FeedUrl,omitempty"`
+	XrayIndex                         *bool                   `json:"xrayIndex,omitempty" xml:"xray>enabled,omitempty"`
+	ListRemoteFolderItems             *bool                   `json:"listRemoteFolderItems,omitempty" xml:"listRemoteFolderItems,omitempty"`
+	EnableTokenAuthentication         *bool                   `json:"enableTokenAuthentication,omitempty" xml:"enableTokenAuthentication,omitempty"`
+	ContentSynchronisation            *ContentSynchronisation `json:"contentSynchronisation,omitempty" xml:"contentSynchronisation,omitempty"`
+	BlockPushingSchema1               *bool                   `json:"blockPushingSchema1,omitempty" xml:"blockPushingSchema1,omitempty"`
+	QueryParams                       *string                 `json:"queryParams,omitempty" xml:"queryParams,omitempty"`
+	PropagateQueryParams              *bool                   `json:"propagateQueryParams,omitempty" xml:"propagateQueryParams,omitempty"`
 }
 
 // ContentSynchronisation represents smart remote repository configuration
 type ContentSynchronisation struct {
```

Comment:
Haha this one plays with my brain 😢 

Specifically for some reason I think it should be `Synchronization` with a `z`.

But Artifactory is likely using the European spelling which I believe uses an `s`.

Reply:
Gah, yeah, JFrog seem quite inconsistent with this as they use a `z` when they refer to synchronization in their [docs](https://www.jfrog.com/confluence/display/JFROG/Smart+Remote+Repositories).